src/Commands/Env/DeployCommand.php

```diff
@@ -31,7 +32,6 @@ public function deploy($site_env, $options = [
                                     'cc' => false,
                                     'updatedb' => false,])
     {
-        // @TODO: Switch this to the standard site.env input format?
```

Comment:
@ronan I don't follow this comment - are you saying this is this not the standard site.env format?